.github/workflows/ci-cd.yml

```diff
@@ -1,35 +1,52 @@
-name: Tests
-on: [merge_group, push, pull_request, workflow_dispatch]
+name: CI/CD
```

Comment:
We need to change link in README as well:
```diff
-    .. image:: https://github.com/vacanza/python-holidays/workflows/Tests/badge.svg
+    .. image:: https://github.com/vacanza/python-holidays/workflows/CI%2FCD/badge.svg
```

Reply:
Yep, missed this one. Updated.